templates/kubernetes/docs/1.25/components.md

```diff
@@ -37,7 +37,7 @@ Other information about this release can be found on the following pages:
 ## What's new
 
 For a list of new features, changes, deprecations, and bug fixes in this
-release, please see the [release notes](release-notes).
+release, please see the [Release notes](release-notes).
```

Comment:
Questionable capitalisation?

Reply:
Yes, this seems to be a regression, will address it across the source next pulse